front/src/common/IntervalsEditor/IntervalsEditorCommonForm.tsx

```diff
@@ -19,51 +20,68 @@ const IntervalsEditorCommonForm = ({
   interval,
   selectedIntervalIndex,
   setData,
+  setSelectedIntervalIndex,
   totalLength,
 }: IntervalsEditorFormProps) => {
   const { t } = useTranslation('common/common');
 
-  /**
-   * Resize segment from inputs
-   */
-  // TODO: fix bug
-  const resizeSegmentByInput = (
-    e: React.ChangeEvent<HTMLInputElement>,
-    context: 'begin' | 'end'
-  ) => {
-    const gap = parseFloat(e.target.value) - interval[context];
-    const { result } = resizeSegment(data, selectedIntervalIndex, gap, context, false);
-    const fixedResults = fixLinearMetadataItems(result.filter(notEmpty), totalLength);
-    setData(fixedResults);
+  const [begin, setBegin] = useState<number>(interval.begin);
+  const [end, setEnd] = useState(interval.end);
+
+  useEffect(() => {
+    setBegin(interval.begin);
+    setEnd(interval.end);
+  }, [interval]);
+
+  /** Resize segment from inputs */
```

Comment:
Is this comment needed ? The function name is pretty clear I think

Reply:
Done !